File: ghettoLog.py
#!/usr/bin/python

import sys;


def ghettoLog(message):
  file = open("/tmp/ghettoLog.txt","a")
  file.write(message+"\n")
  file.close()


print("Writing to ghetto log")
cmdLineArgsAsString = ";".join(sys.argv)
ghettoLog("command line args are "+cmdLineArgsAsString)

File: ghettoLog.pl
#!/usr/bin/perl

use strict;
use warnings;

sub ghettoLog #message
{
  my $message = $_[0];
  if (defined $message) {
    my $outfile = "/tmp/ghettoLog.log";
    open(FILE,">> $outfile") || die "problem opening $outfile\n";
    print FILE "$message\n";
    close(FILE);
  }
}


ghettoLog("using sub 2");
ghettoLog("another sub 3");


